### PR TITLE
Fixes #10915 - Update using module-qualified names

### LINKED ISSUE #10915: Document module qualified invocation
### Prerequisites

- [X] **Existing Issue:** Search the existing issues for this repository. If there is an issue that fits your needs do not file a new one. Subscribe, react, or comment on that issue instead.
- [X] **Descriptive Title:** Write the title for this issue as a short synopsis. If possible, provide context. For example, "Document new `Get-Foo` cmdlet" instead of "New cmdlet."

### PowerShell Version

5.1, 7.2, 7.3, 7.4

### Summary

```
[ModuleName]\[command name]

[Module path with forward slashes]\[command name]
```

This format will trigger modules to be loaded if auto-loading is enabled.

### Details

_No response_

### Proposed Content Type

Other / Unknown

### Proposed Title

_No response_

### Related Articles

_No response_

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Update using module-qualified names

- Fixes [AB#226693](https://dev.azure.com/msft-skilling/cebd7ef5-4282-448b-9701-88c8637581b7/_workitems/edit/226693)
- Fixes #10915

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Command_Precedence.md [hunk 1]
```diff
@@ -145,15 +145,27 @@ For cmdlets, you can use the module-qualified name. For executables, you can
...
 `where` use `where.exe`.
 
-### Using module-qualified names
+### Use module-qualified names
 
 Using the module-qualified name of a cmdlet allows you to run commands hidden
 by an item with the same name. For example, you can run the `Get-Date` cmdlet
-by qualifying it with its module name **Microsoft.PowerShell.Utility**.
+by qualifying it with its module name **Microsoft.PowerShell.Utility** or full
+path. When you use module-qualified names, the module is automatically imported
```
Comment (TravisEz13) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Command_Precedence.md:153]:
```suggestion
path. When you use module-qualified names, the module is automatically imported if `$psModuleAutoLoadingPreference` is set to `all`
```

Comment (TravisEz13) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Command_Precedence.md:153]: we could say, based of `$psModuleAutoLoadingPreference` and link to here:
https://github.com/MicrosoftDocs/PowerShell-Docs/blob/6feaff6bce4a5135b83030dbbc3a4393ab3316b0/reference/5.1/Microsoft.PowerShell.Core/About/about_Preference_Variables.md?plain=1#L1092

**Review by TravisEz13 — COMMENTED** [review]
I think it looks good other than the one comment.

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Command_Precedence.md (hunk 1 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Command_Precedence.md:153]: Added

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**